src/replaceRuleSelector.js

```diff
@@ -33,7 +48,9 @@ function explodeSelector(selector, options) {
 
       let newParts
       if (postSelectors.length === 0) {
-        newParts = bodySelectors.map((s) => preWhitespace + pre + s)
+        newParts = bodySelectors.map((s) => {
+          normalizeSelector(s, preWhitespace, pre)
```

Comment:
I would like to have a better naming because `pre` is not actually `pre` sometimes.